dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -94,6 +94,9 @@ The index file can be generated using a dedicated script::
 
     ``<path_to_dali>/tools/wds2idx.py <path_to_archive> <path_to_index_file>``
 
+Should the index file be not provided, it will be automatically inferred from the tar file.
```

Comment:
```suggestion
If the index file is not provided, it will be automatically inferred from the tar file.
```
Maybe just for me but it reads better.